packages/pipeline-editor/src/PipelineController/index.ts

```diff
@@ -519,6 +519,51 @@ class PipelineController extends CanvasController {
     return "";
   }
 
+  getPropertyValue(info: any, value: any, label?: string): any {
+    if (info.data?.format === "inputpath" || info.format === "inputpath") {
+      // Find the node corresponding to the input node
+      const upstreamNode = this.findExecutionNode(value?.value ?? "");
+      const upstreamNodeLabel = upstreamNode?.app_data?.ui_data?.label;
+      const upstreamNodeDef = this.getAllPaletteNodes().find(
+        (nodeDef) => nodeDef.op === upstreamNode?.op
+      );
+      // Add each property with a format of outputpath to the options field
+      const upstreamNodeOption = upstreamNodeDef?.app_data?.properties?.uihints?.parameter_info?.find(
+        (prop) => {
+          return prop.parameter_ref === value.option;
+        }
+      )?.label?.default;
+      return {
+        label: label ?? info.label.default,
+        value: upstreamNodeLabel
+          ? `${upstreamNodeLabel}: ${upstreamNodeOption ?? ""}`
```

Comment:
if I'm reading this right the `:` will always be shown here, it should only show if `upstreamNodeOption` exists (see `getLabel()` in `NestedEnumControl` for reference